cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2256,19 +2323,14 @@ ConfigMap getKafkaAncialiaryCm()    {
 
             ConfigMap brokerCm = kafkaCluster.generateAncillaryConfigMap(loggingCm, kafkaExternalAdvertisedHostnames, kafkaExternalAdvertisedPorts);
 
-            String brokerConfiguration = brokerCm.getData().get(KafkaCluster.BROKER_CONFIGURATION_FILENAME);
-            brokerConfiguration += brokerCm.getData().getOrDefault(KafkaCluster.BROKER_ADVERTISED_PORTS_FILENAME, "");
```

Comment:
How are you handling changes to `BROKER_ADVERTISED_PORTS_FILENAME` and  `BROKER_ADVERTISED_HOSTNAMES_FILENAME ` if not through the hash?

Reply:
reverted